### PR TITLE
PFM-4118-added worflow businessService name and must roll host

### PR DESCRIPTION
**Pull request by shubhang-eGov**

### CODE REVIEW

File: deploy-as-code/helm/environments/works-dev.yaml [hunk 1]
```diff
@@ -144,6 +144,7 @@ contract-service:
   contract-search-max-limit: 50
   works-contract-service-code: "WORKS-CONTRACT"
   contract-workflow-business-service: "CONTRACT"
+  contract-workflow-revision-business-service: "WORKORDER-REVISION"
```
Comment (subhashini-egov) [inline deploy-as-code/helm/environments/works-dev.yaml:147]: @shubhang-eGov keep the nomenclature consistent. Use CONTRACT-REVISION or REVISED-CONTRACT.

File: deploy-as-code/helm/charts/digit-works/backend/contracts/values.yaml [hunk 2]
```diff
@@ -109,6 +109,11 @@ env: |
       configMapKeyRef:
         name: egov-service-host
         key: egov-location
+  - name: WORKS_MUSTER_ROLL_HOST
```
Comment (subhashini-egov) [inline deploy-as-code/helm/charts/digit-works/backend/contracts/values.yaml:112]: We should not be tightly coupling contract service with muster roll. @jagankumar-egov - we are getting more and more services tightly coupled with the contract service. We need a BFF pattern for this.

**Review by subhashini-egov — CHANGES_REQUESTED** [review]

File: deploy-as-code/helm/environments/works-dev.yaml (hunk 1 above)
Comment (shubhang-eGov) [reply deploy-as-code/helm/environments/works-dev.yaml]: done

File: deploy-as-code/helm/charts/digit-works/backend/contracts/values.yaml (hunk 2 above)
Comment (subhashini-egov) [reply deploy-as-code/helm/charts/digit-works/backend/contracts/values.yaml:112]: This needs to be addressed in a later release as technical debt. Had a discussion with @shubhang-eGov and @jagankumar-egov about this. UI team to come up with a BFF proposal.

**Review by subhashini-egov — APPROVED** [review]

**Pull request merged**